reference/release-notes/1.22.0.md

```diff
@@ -0,0 +1,83 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.22.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+The 1.22.0 release of Anbox Cloud brings the following features and improvements:
+
+### Core stack improvements
+* Support for 64-bit only ARM systems for all supported Android versions.<!--AC-1890, AC-2428-->
```

Comment:
The `64-bit only` system can refer to `x86_64` arch as well. I'd suggest dropping `ARM` here.